app/jobs/reset_user_data_usage.py

```diff
@@ -0,0 +1,32 @@
+from datetime import datetime
+
+from app import logger, scheduler
+from app.db import get_users, get_db,  crud
+from app.models.user import  UserDataLimitResetStrategy
+
+reset_strategy_to_days = {
+    UserDataLimitResetStrategy.day.value :1,
+    UserDataLimitResetStrategy.week.value: 7,
+    UserDataLimitResetStrategy.month.value : 30,
+    UserDataLimitResetStrategy.year.value : 365,
+}
+
+def reset_user_data_usage():
+    now = datetime.utcnow().timestamp()
```

Comment:
you must remove `.timestamp()` to prevent error below in following lines
`TypeError: unsupported operand type(s) for -: 'float' and 'datetime.datetime'`